outputs/core_pools.json

```diff
@@ -46,9 +47,11 @@
     "0x2ce4457acac29da4736ae6f5cd9f583a6b335c270000000000000000000004dc": "sFRAX/4POOL",
     "0x2d6ced12420a9af5a83765a8c48be2afcd1a8feb000000000000000000000500": "cbETH/rETH/wstETH",
     "0x315dd595e82bdc0c194f3a38a08fde480d7e5d2100020000000000000000056a": "ECLP-wUSDM-GYD",
+    "0x36c2f879f446c3b6533f9703745c0504f3a84885000200000000000000000591": "ECLP-sFRAX-aFRAX",
     "0x3fd4954a851ead144c2ff72b1f5a38ea5976bd54000000000000000000000480": "ankrETH/wstETH-BPT",
     "0x46472cba35e6800012aa9fcc7939ff07478c473e00020000000000000000056c": "ECLP-GHO-AUSDC",
     "0x4b3af34eb1135d59df8b9cdc2ff07d30d05334c400000000000000000000050d": "weETH/rETH",
+    "0x59743f1812bb85db83e9e4ee061d124aaa64290000000000000000000000052b": "sUSDe/sFRAX",
```

Comment:
New pool as of yesterday, unsure why it was not caught uy the old subgraph.  maybe no sUSDe?